lib/fs.js

```diff
@@ -1838,7 +1842,9 @@ if (isWindows) {
   nextPart = function nextPart(p, i) {
     for (; i < p.length; ++i) {
       const ch = p.charCodeAt(i);
-      if (ch === 92/*'\'*/ || ch === 47/*'/'*/)
+      const isSeparator = ch === CHAR_BACKWARD_SLASH ||
+                          ch === CHAR_FORWARD_SLASH;
+      if (isSeparator)
```

Comment:
Nit: the extra variable could also just be replaces with a comment.

Reply:
@BridgeAR done